massa-consensus/src/settings.rs

```diff
@@ -157,7 +159,14 @@ pub struct ConsensusSettings {
 }
 
 impl ConsensusSettings {
+    /// Utility method to derivate a full configuration (static + user defined) from an user defined one
+    /// This is really handy in tests where we want to mutate static defined values
     pub fn config(&self) -> ConsensusConfig {
+        // TODO: these assertion should be checked at compile time
```

Comment:
Do you have a plan to do that ? Anyway you should open and link an issue here

Reply:
The link bellow reference an issue in https://github.com/rust-lang that the one we should track

Reply:
I was more thinking of a link to a massa issue (that can be one that link this issue)

Reply:
I think since the issue is not on our side it's purpose-less to open a massa issue, the problem is already tracked in the right place, where it could be solved :)

Reply:
I was thinking about something on our side to not forget to track what's happening upstream

Reply:
And I think it's a bit pointless

Reply:
And I don't, we are forgetting so much stuff every day